README.md

```diff
@@ -51,6 +51,18 @@ Commands:
 
 ---
 
+### Caching
+
+You can use caching by including this module
+
+    include Gris::CacheKey
+
+in your ActiveRecord models, and then cache inside GET requests like this
+
+    offer_event =  Gris.cache.fetch(OfferEvent.cache_key_for(id)) do
+      OfferEvent.find(id)
+    end
```

Comment:
Are you assuming this would be based on the object ID, and not have additional parameters such as an `expires_in` value?